frontend/src/views/projectStats.js

```diff
@@ -40,12 +41,9 @@ export function ProjectStats() {
       // To fix: set this URL with an ENV VAR later
       if (defaultComment.length) {
         fetchExternalJSONAPI(
-          `https://osm-stats-production-api.azurewebsites.net/stats/${defaultComment[0].replace(
-            '#',
-            '',
-          )}`,
+          `${OHSOME_STATS_BASE_URL}/stats/${defaultComment[0].replace('#', '')}?ohsomeFormat=false`,
```

Comment:
also here the `ohsomeFormat=false` parameter will have no effect.

Reply:
Thank you for pointing out the unnecessary parameter!